index.html

```diff
@@ -238,5 +235,76 @@ <h3>Multi-Post Stories</h3>
         </ul>
       </section>
     </main>
+    <footer>
+      <div class="contact-text">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form
+        action="https://formspree.io/f/xbjwlnpa"
+        method="post"
+        class="formar"
+      >
+        <ul>
+          <li id="user-name">
+            <input type="text" placeholder="Text input" maxlength="30" />
```

Comment:
- [ ] Well done including this input. I think you forgot to pass in the `required` attribute that will validate that the input field has some content before any submission is made.